### PR TITLE
Pathfinding: use rolling stop maximum speed and length

### LINKED ISSUE #8154: core: pathfinding: add rolling stock max speed limit to the api inputs
The pathfinding api only takes rolling stock inputs that are directly related to the paths that can be used. 

But now that we find the fastest path, the rolling stock max speed and length are relevant inputs. Currently we have generic values of 320km/h and 0m, which may be good enough for now. But we end up with weird magic values in the code. 

See https://github.com/OpenRailAssociation/osrd/pull/8145#discussion_r1687635160

### PR DESCRIPTION
**Pull request by Sh099078**
Fixes #8154.

### CODE REVIEW

File: core/src/main/kotlin/fr/sncf/osrd/api/api_v2/pathfinding/PathfindingBlocksEndpointV2.kt [hunk 1]
```diff
@@ -94,7 +93,7 @@ fun runPathfinding(
         )
 
     // TODO: add the rolling stock global speed limit to the request
```
Comment (Erashin) [inline core/src/main/kotlin/fr/sncf/osrd/api/api_v2/pathfinding/PathfindingBlocksEndpointV2.kt]: Remove TODO, you just fixed it :)

**Review by Erashin — APPROVED** [review]
Core part looks good to me, gj :)

File: editoast/src/views/v2/path/pathfinding.rs [hunk 2]
```diff
@@ -54,6 +55,12 @@ struct PathfindingInput {
     rolling_stock_supported_signaling_systems: Vec<String>,
     /// List of waypoints given to the pathfinding
     path_items: Vec<PathItemLocation>,
+    /// Rolling stock maximum speed
+    #[schema(value_type=f64)]
+    rolling_stock_maximum_speed: OrderedFloat<f64>,
+    /// Rolling stock length
+    #[schema(value_type=f64)]
```
Comment (leovalais) [inline editoast/src/views/v2/path/pathfinding.rs]:
```suggestion
    #[schema(value_type = f64)]
    rolling_stock_maximum_speed: OrderedFloat<f64>,
    /// Rolling stock length
    #[schema(value_type = f64)]
```

**Review by leovalais — APPROVED** [review]
LGTM, congratz! 🚀

**Review by Erashin — COMMENTED** [review]
Some e2e tests are failing though, will need looking into.

**Comment by Khoyo** [comment]
> Some e2e tests are failing though, will need looking into.

Yeah, looking at the videos they seem to have a really low timeout. We should consider increasing it at least until osrdyne is able to correctly report core readiness.

**Review by clarani — APPROVED** [review]
LGTM :white_check_mark: tested

**Pull request merged**